### ISSUE TITLE
We're doing the same thing

### ISSUE THREAD

**Issue by tony** [rockandska/pytest-tmux#1]
@rockandska 

_Air traffic control / collision avoidance activated 😄_ 

i've made a [pytest-tmux](https://github.com/tmux-python/pytest-tmux) repo and reserved it on PyPI. In anticipation of having this moved into the package: https://libtmux.git-pull.com/pytest-plugin/index.html. I was going to publish it **today**. I've hit the breaks.

I want to show you something - this is from July 17th, me preparing to do a pytest plugin: https://github.com/tmux-python/libtmux/issues/381

but you've also told me you want to do make a pytest-plugin for tmux. I don't recall if it's before or after. 

What do you propose? What do you think?

If this really seems like something you've been building up to do and can follow through with it, I'm open to transitioning ownership of the pypi package to you (and I'll have to pick another). I note: That's easy to do now - since I've hit the breaks - but harder to do later 😆

**Comment by tony** [comment]
@rockandska I found the [develop](https://github.com/rockandska/pytest-tmux/tree/develop) branch

Now I'm more convinced you have a creative vision for this.

If you want to talk off tracker, ping or email me.  tony at git-pull dot com.  no biggie for me

**Comment by rockandska** [comment]
> I was going to publish it today. I've hit the breaks.

Thanks for the consideration

> I want to show you something - this is from July 17th, me preparing to do a pytest plugin: https://github.com/tmux-python/libtmux/issues/381

yeah, I remember discuss it and have seen it

> but you've also told me you want to do make a pytest-plugin for tmux. I don't recall if it's before or after.

Does it really matter ? ^^

> What do you propose? What do you think?

> If this really seems like something you've been building up to do and can follow through with it, I'm open to transitioning ownership of the pypi package to you (and I'll have to pick another). I note: That's easy to do now - since I've delayed publishing after finding we're on the same path - but harder to do later laughing

Not sure if I would have the time and skill to maintain it, but I'm inclined to co-author it.
I could even let you have this repository and clean it up if you think that it will fit better the users requirements.

> Now I'm more convinced you have a creative vision for this.

Yeah we clearly not have the same vision / goal :)
I need to push my last modifications, but I need to refactor some of the logic.
If you are not in rush, maybe I could try to push some examples to show you a better vision of what I try to achieve
Actually, I use [jhawthorn/ttytest](https://github.com/jhawthorn/ttytest) and was my first motivation to have a similar thing in pytest.

> If you want to talk off tracker, ping or email me. tony at git-pull dot com. no biggie for me

Sure thing, we could even chat on gitter if it is not a problem for you

**Comment by tony** [comment]
> Does it really matter ? ^^

yes of course we don't want to step on anyone's toes in open source

> Not sure if I would have the time and skill to maintain it, but I'm inclined to co-author it.
> I could even let you have this repository and clean it up if you think that it will fit better the users requirements.

We have various options

As it stands, neither of us have a 1.0 package, so APIs are free to be shifted around, things can be changed.  Nobodies using the package - so ample time to try things out

In regards on what you plan to push - I look forward to seeing your changes

Thanks for telling me about [jhawthorn/ttytest](https://github.com/jhawthorn/ttytest)

In regards to gitter, how is slack for you? (not that you'd need to be on it regularly). if you did, I'd at least need you to send me your email to invite you
